reference/constraints/Collection.rst

```diff
@@ -261,9 +263,9 @@ you can do the following:
 
         class Author
         {
-            protected $profileData = ['personal_email'];
```

Comment:
?

Reply:
I think it was wrong at first, it should have been `protected array $profileData = ['personal_email' => 'email@example.com'];`, like in the attributes example. But I saw that plenty examples of `loadValidatorMetadata` used ellipsis in place of properties declarations. So I removed it.

Should it be reverted and updated to `protected array $profileData = ['personal_email' => 'email@example.com'];` ?

Reply:
Ok lets remove it